src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -21,6 +22,7 @@ const SheetAddTopic = ({refTopic, onAdd, topics, onClose, chatTopics}) => {
   const rbSheetHeight = 330;
   const [, setWidthInput] = React.useState(0);
   const inputRef = React.useRef();
+  const dynamicColor = useDynamicColors(isAnonimity);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useDynamicColors` hook is being used with `isAnonimity` as an argument. This change is not mentioned in the PR summary, so it's unclear what the expected behavior is. It would be helpful to know how this affects the component's rendering and whether there are any potential side effects.
```diff
+ const dynamicColor = useDynamicColors(isAnonimity);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->